<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringBuilder::WidgetInsertTextSetting</class>
 <widget class="QWidget" name="StringBuilder::WidgetInsertTextSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="HistoryComboBox" name="combox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="maxCount">
      <number>24</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="StringBuilder::WidgetPositionFixer" name="widgetPositionFixer">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StringBuilder::WidgetPositionFixer</class>
   <extends>QFrame</extends>
   <header>stringbuilder/widgets/widgetpositionfixer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HistoryComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/historycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
